src/pegout/components/PegoutForm.vue

```diff
@@ -386,6 +387,7 @@ export default defineComponent({
         .finally(() => {
           loadingQuotes.value = false;
           showStep.value = true;
+          loading.value = false;
```

Comment:
Is possible to reuse the existing `loadingQuotes`?

Reply:
I tried to reused but it shows the seachign options from the beginning, that's why I decided to create a different one

Reply:
ok, you got it!